tests/test_pyoidc_facade.py

```diff
@@ -46,7 +44,15 @@ def test_is_registered(self):
     @responses.activate
     def test_register(self):
         registration_endpoint = self.PROVIDER_BASEURL + '/register'
-        responses.add(responses.POST, registration_endpoint, json=self.CLIENT_METADATA.to_dict())
+        client_registration_response = {
+            'client_id': 'client1',
+            'client_secret': 'secret1',
+            'client_name': 'Test Client',
+            'redirect_uris': ['https://rp.example.com/redirect_uri'],
+            'registration_client_uri': 'https://op.example.com/register/client1',
+            'registration_access_token': 'registration_access_token1'
+        }
+        responses.add(responses.POST, registration_endpoint, json=client_registration_response)
```

Comment:
Why this change?

Reply:
To mock client registration response.

Reply:
What was the difference needed for mocking the response like that instead of using the existing `CLIENT_METADATA`?

Reply:
The response must have `redirect_uris`.

Reply:
The response still has to be mocked to initialize `ClientMetadata` by using the parameters taken from the response. Then only it can be considered that `assert facade.is_registered() is True` is actually true.

Reply:
> Then only it can be considered that assert facade.is_registered() is True is actually true.

Not sure I'm following - the tests passed before this change and the actual values in the response does not matter.
What makes this change necessary for this tests?